### PR TITLE
Replace bash code with proper python code

### PR DESCRIPTION
**Pull request by rvais**
Solves issue #12

### CODE REVIEW

File: java-utils/mvn_build.py [hunk 1]
```diff
@@ -188,12 +190,14 @@ def goal_callback(option, opt_str, value, parser):
...
-        if [ -f .xmvn-builddep ]; then
-            echo -----BEGIN MAVEN BUILD DEPENDENCIES-----
-            gzip -9nc <.xmvn-builddep | base64
-            echo -----END MAVEN BUILD DEPENDENCIES-----
-        fi
-        """, shell=True, env=os.environ).wait()
-
-    sys.exit(p.returncode)
+    fname = ".xmvn-builddep"
+    if (os.path.exists(fname) and os.path.isfile(fname)):
```
Comment (mizdebsk) [inline java-utils/mvn_build.py]: If a file is regular file then it must exist, so `os.path.isfile(fname)` should be enough.

**Comment by xvaisr** [comment]
I created a test case but unfortunately it works only for python 3. For python 2 it fails due to differences in gzip module between python versions. I will redo it after I figure out how to make it produce the same output in case of python 2.

**Comment by rvais** [comment]
I created a test case but unfortunately it works only for python 3. For python 2 it fails due to differences in gzip module between python versions. I will redo it after I figure out how to make it produce the same output in case of python 2.

**Comment by msimacek** [comment]
Almost a year without any activity. Closing. Feel free to reopen

**Comment by rvais** [comment]
Yeah. I completely forgot about this old issue since I left the team and started to work for Messaging QE ... If I find a bit of time I will finish it. I have more experience with python now, which should help.

**Pull request closed without merging**